src/people.ts

```diff
@@ -0,0 +1,7 @@
+const API = 'https://mate-academy.github.io/react_people-table/api/people.json';
+
+export const getPeople = async () => {
+  const response = await fetch(API);
```

Comment:
Find a mother and a father by motherName and fatherName and add them to the person for future use. You can highlight mother/father on click